solutions/instances/variables.tf

```diff
@@ -124,7 +124,7 @@ variable "add_bucket_name_suffix" {
 
 variable "cos_region" {
   type        = string
-  default     = "us-south"
+  default     = null
   description = "The Cloud Object Storage region."
```

Comment:
Update the description to say that is no value pass it default to the value of the 'region input variable.

Reply:
Done